lib/precompiled/05-modexp.js

```diff
@@ -30,7 +30,7 @@ function getAdjustedExponentLength (E, eLen) {
 
 // Taken from https://stackoverflow.com/a/1503019
 function expmod (B, E, M) {
```

Comment:
This function should use the reduction context in bn.js for proper speed.

Reply:
I'm not a genius on this, but what I extracted from the comments around the stack overflow solution, I think this actually is a reduction context (correct me if I'm wrong, I have no idea what a reduction context actually really is. :-))

I initially wanted to use the bn.js reduction context after reading your comments on the initial PR, but I didn't figure out how to apply this to this special problem. I actually implemented the stack overflow solution after running into long running tests ending with out-of-memory errors, with the stackoverflow solution this went back to be pretty snappy then.

You can rewrite this, but please make sure before, that you're not replacing one valid solution with another, this actually took me quite some time to implement.